src/main/kotlin/org/pkl/lsp/ast/Expr.kt

```diff
@@ -400,15 +401,42 @@ class PklLogicalNotExprImpl(
   }
 }
 
-class PklAdditiveExprImpl(
+abstract class PklBinExprImpl(
   override val project: Project,
   override val parent: PklNode,
   override val ctx: TreeSitterNode,
```

Comment:
Let's add `override val operator: PklNode` as a constructor arg here, since we've already found the node when creating this tree.